recipe/bld.bat

```diff
@@ -1,22 +1,42 @@
-robocopy . "%PREFIX%" /S /XF bld.bat
-if errorlevel NEQ 1 exit 1
+set "PREFIX_NSIS=%PREFIX%\NSIS"
+robocopy . "%PREFIX_NSIS%" /S /XF bld.bat Docs Examples
```

Comment:
So we had been dumping things into `%PREFIX%` directly. Have no problem changing it for 3.x. How does `conda` make sure this stuff gets added to the `%PATH%` though?

Reply:
Just mirrored the behavior from the Anaconda recipe.
This is simply what `constructor` expects, see: https://github.com/conda/constructor/blob/b2891730f774b97e646a80476656e29eaca2193a/constructor/winexe.py#L26
Well, with this is won't be on `PATH` 😕 ... What do think would be the best way to also make sure it's accessible via `PATH`? I think dumping all its subfolders into one dedicated `NSIS` folder is a good thing, but having its `.exe` files available on `PATH` would be nice, too.

Reply:
Can think of a couple of solution.

One would be an activate script that adds these directories to the path. This is probably the most effective and activate scripts are pretty reliable. Though it requires that people actually activate the environment for them to work.

Another would be placing the executables in both locations. Not sure how library loading proceeds here for the executables should they need them.

Reply:
Placing them in other locations won't work since NSIS requires that directory layout. I was thinking if symlinks were possible, but having thought about it, I don't think we should use those, esp. on Windows.
Activate script it is! (And overall we shouldn't really care if people don't activate the environments beforehand since that's just not how Conda should be used :wink:)